server/utils/nunjucksSetup.ts

```diff
@@ -51,14 +51,7 @@ export function registerNunjucks(app?: express.Express): Environment {
     },
   )
 
-  njkEnv.addFilter('initialiseName', (fullName: string) => {
-    // this check is for the authError page
-    if (!fullName) {
-      return null
-    }
-    const array = fullName.split(' ')
-    return `${array[0][0]}. ${array.reverse()[0]}`
-  })
+  njkEnv.addFilter('initialiseName', initialiseName)
```

Comment:
Moving this filter to be a function in `utils.ts` (in line with how the template app now is) so it can be used in main code as well as Nunjucks files.